lib/linkerd2-proxy-resolve/src/destination/resolution.rs

```diff
@@ -165,10 +165,9 @@ where
                 .expect("resolution should not have been dropped");
 
             let query = self.query.take().expect("invalid state");
-            let ctx = logging::Section::Proxy.bg(LogCtx(query.authority().clone()));
 
-            let daemon = ctx.future(Daemon { updater, query });
-            tokio::spawn(Box::new(daemon));
+            // todo: set logging context within the daemon task.
+            task::spawn(Daemon { updater, query });
```

Comment:
This should be something like:
```rust
let daemon = Daemon { updater, query }
    .instrument(info_span!("resolve", authority = %query.authority())
task::spawn(daemon)
```
(assuming that the authority implements `fmt::Display`, which IIRC it does)

the imports to make this work will be something like
```rust
use tracing::info_span;
use tracing_futures::Instrument;
```
```